src/panels/lovelace/cards/hui-button-card.ts

```diff
@@ -145,13 +145,9 @@ export class HuiButtonCard extends LitElement implements LovelaceCard {
 
     if (this._config.entity && !stateObj) {
       return html`
-        <hui-warning
-          >${this.hass.localize(
-            "ui.panel.lovelace.warning.entity_not_found",
-            "entity",
-            this._config.entity
-          )}</hui-warning
-        >
+        <hui-warning>
+          ${createEntityNotFoundWarning(this.hass, this._config.entity)}
+        </hui-warning>
```

Comment:
Why actually expect caller to still wrap it in `hui-warning` and not just return a template from `createEntityNotFoundWarning` ?